playbooks/ims-collection-sample.yaml

```diff
@@ -0,0 +1,278 @@
+###############################################################################
```

Comment:
As discussed this file was accidentally copied over from `tests` directory and should be deleted. I would suggest deleting the `playbooks` directory and it's contents altogether and then we can revert the `.gitignore` file change above.